docker/tomcat_expedius/conf/templates/expedius.properties

```diff
@@ -0,0 +1,97 @@
+##----------------------------------------------------------------------
+## Expedius Properties 												 
+## Author: Dennis Warren @ Colcamex Resources
+## Contains the configuration settings for Expedius								
+## Date: February 2012 
+## Updated: January 2020														
+##----------------------------------------------------------------------
+
+## Current version.
+VERSION=Version 4.12.19
+
+## Vendor to contact for support.
+VENDOR=<a href="http://www.colcamex.com/" target="_blank" >Support and distribution by Colcamex Resources Inc.</a> 
+
+## Settings for which services to run. Values are true or false.
+EXCELLERIS=true
+IHAPOI=false
+
+## settings for email server. If EMAIL_ON is yes then a smtp server, such as postfix, 
+## is expected to be installed on this server.
+EMAIL_ON=
+ADMIN_EMAIL=
+EMAIL_HOST=localhost
+
+## Install directory for your Tomcat Instance. Normally should be /var/lib/tomcat6/webapps [Expedius].
+TOMCAT_ROOT=/usr/local/tomcat/webapps
+
+## Context path. This can be adjusted if, for whatever reason, the root folder name for Expedius is changed.
+CONTEXT_PATH=/Expedius
+
+## the Expedius log path can be set to suite your environment. Usually /var/log
+LOG_PATH=/var/lib/expedius/logs/
+
+## URL for the webservice endpoint which will be handling the downloaded HL7 lab file. For Oscar the 
+## path will be something like this: https://127.0.0.1:8443/oscar_12/ws
+
+EMR_SSL_ENABLED=false
+EMR_CONTEXT_PATH=
+EMR_HOST_NAME=
+EMR_SERVICE_ENDPOINT=ws
+EMR_LOGIN_ENDPOINT=LoginService
+EMR_LAB_UPLOAD_ENDPOINT=LabUploadService
+
+## DEFAULT Username and Password for EMR webservices.
+EMR_WS_USERNAME=
+EMR_WS_PASSWORD=
+SERVICE_NUMBER=
+
+# service number: like a provider number id but for this service.
+SERVICE_NAME=Expedius
+
+## Java Trust Store password.  Ie.: cacerts
+## This is used to confirm an EMR's unsigned certificate.
+CACERTS_PASSWORD=changeit
+
+## Trust Certificate and protocol settings. Do not change unless you know for sure...
+## If password protected, make it the same as the keystore.
+TRUSTSTORE_URL=/var/lib/expedius/.ssl/expedius_trust.jks
+TRUSTSTORE_ALIAS=excellerisTrust
+KEYSTORE_URL=/var/lib/expedius/.ssl/expedius_key.jks
+KEYSTORE_ALIAS=excellerisKey
+
+# CA Certificate path and password
+STORE_PASS=3mr1esting89!
```

Comment:
Does this need protected? is it specific to the key you generated and committed?

Reply:
It's a password for the jks truststore. This should be generated randomly as well.  I left it for purposes of testing.